lsp-mode.el

```diff
@@ -4898,9 +4898,9 @@ Shown after the code action in `lsp-execute-code-action',
         (disabled?
          (concat
           (propertize title 'face 'lsp-disabled-code-action-face)
-          " "
```

Comment:
Do we need this commit at all? I don't know how far you want to go to support spec-breaking LS.

Reply:
Sorry, I got lost. Can you elaborate? There are several improvements in this PR so IMHO better to have them in.

Reply:
Code actions can have a `disabled?` field, which, as per the spec, *must* have a `reason` member. This commit handles the case where language servers send invalid code actions that have a `disabled` field without a `reason` (I don't know of an LS that does that, though). This is just being defensive, and I'm unsure whether we need to handle that case (it is a spec violation after all).

Reply:
Most of the time we don't handle them. Leaving up to you to decide.

Reply:
I propose to keep it in then, because it can't hurt to have a more lenient lsp implementation.